src/routes/scope-selection/ScopeSelection.tsx

```diff
@@ -0,0 +1,119 @@
+  padding: ${props => props.theme.spacings.xs} 0;
+  background-color: ${props => props.theme.colors.background};
+`
+
+const StyledPressable = styled.Pressable`
+  padding: ${props => props.theme.spacings.sm};
+  border-bottom: 1px solid ${props => props.theme.colors.disabled};
+`
+
+const DisciplineContainer = styled.View`
+  margin: 0 ${props => props.theme.spacings.sm};
+`
+
+const HighlightContainer = styled.Text<{ disabled: boolean }>`
+  flex-direction: row;
+  color: ${props => (props.disabled ? props.theme.colors.disabled : props.theme.colors.black)};
+`
+
+const highlightText = (textArray: [string] | [string, string, string], disabled: boolean): JSX.Element => (
+  <HighlightContainer disabled={disabled}>
+    <ContentTextLight>{textArray[0]}</ContentTextLight>
+    <ContentTextBold>{textArray[1]}</ContentTextBold>
+    <ContentTextLight>{textArray[2]}</ContentTextLight>
+  </HighlightContainer>
+)
+
+type ScopeSelectionProps = {
+  queryTerm: string
+  setQueryTerm: (newString: string) => void
+  showSearchResults: boolean
+  setShowSearchResults: (shouldShow: boolean) => void
+  navigateToDiscipline: (item: Discipline) => void
+  navigateToManageSelection: () => void
+  selectedProfessions: number[] | null
+}
+
+const ScopeSelection = ({
+  queryTerm,
+  setQueryTerm,
+  showSearchResults,
+  setShowSearchResults,
+  navigateToDiscipline,
+  navigateToManageSelection,
+  selectedProfessions,
+}: ScopeSelectionProps): JSX.Element => {
+  const { data: disciplines, error, loading, refresh } = useLoadDisciplines({ parent: null })
+  const allProfessions = useLoadAllDisciplines()
+    .data?.filter(discipline => discipline.total_discipline_children === 0)
+    .map(discipline => formatDiscipline(discipline, {}))
+  const filteredProfessions = useMemo(() => searchProfessions(allProfessions, queryTerm), [allProfessions, queryTerm])
+  const theme = useTheme()
+
+  const disciplineItems = disciplines?.map(item => (
+    <DisciplineListItem key={item.id} item={item} onPress={() => navigateToDiscipline(item)} hasBadge={false} />
+  ))
+
+  return (
+    <>
+      <SearchContainer>
+        <SearchBar
+          query={queryTerm}
+          setQuery={input => {
+            setQueryTerm(input)
+            setShowSearchResults(input.length > 0)
+          }}
+          placeholder={getLabels().scopeSelection.searchProfession}
+          style={{
+            backgroundColor: theme.colors.background,
+          }}
+        />
+      </SearchContainer>
+      {showSearchResults ? (
+        <ScopeContainer>
+          {filteredProfessions?.map(profession => {
+            const disabled = !!selectedProfessions?.includes(profession.id)
+            return (
+              <StyledPressable
+                key={profession.id}
+                onPress={async () => {
+                  await pushSelectedProfession(profession.id).then(() => {
+                    navigateToManageSelection()
+                  })
```

Comment:
:wrench: Please avoid mixing `async/await` and `then`. 
```suggestion
                onPress={async () => {
                  await pushSelectedProfession(profession.id)
                  await navigateToManageSelection()
                })
```